packages/i18n/src/locales/en.json

```diff
@@ -3,6 +3,7 @@
     "download": "Download",
     "download-toast-content": "{{artist}} - {{title}} has been added to downloads.",
     "download-toast-title": "Album added to downloads",
+    "error-load": "Ops, could not load this album. Try later :(",
```

Comment:
Please change the message to "Sorry, this album could not be loaded. Try again later or open a new issue on Github"